index.html

```diff
@@ -269,7 +269,7 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
           </div>
         </div>
       </section>
-      <section class="contact-card-section">
+      <form class="contact-card-section">
         <div class="all-components">
           <div class="tonic-content">
```

Comment:
- [ ] Please, consider that you should add `method` and `action` for your form - that is why your form is not submitted successfully with valid input values.

> 1. Action value should be similar to https://formspree.io/f/{form_id}
> 2. method value equals POST